### ISSUE TITLE
Sunlight as a npm module

### ISSUE THREAD

**Issue by WilliamHuey** [tmont/sunlight#8]
As of now, Sunlight is only a client side library. Would there be a possibility to convert Sunlight to run on node?

Highlightjs is a very popular module, but Sunlight does a much better job at highlighting.

**Comment by tmont** [comment]
The [2.0 branch](https://github.com/tmont/sunlight/tree/2.0) is my progress in turning this into a node module. As you can tell from the commit history, I don't have a lot of time to devote to it at the moment, but I work on it here and there. It was also an opportunity to write _actual_ tests, as well. You know, like a real programmer would :)

I would welcome in and all contributors if they wanted to help.

In short, it's in progress, but progress is slow.

**Comment by WilliamHuey** [comment]
Thanks for the info. I am not going to rush you, just wanted to know if there was progress.
